tests/OTIOCompositionTests.c

```diff
@@ -2058,7 +2058,7 @@ static void otio_nesting_deeply_nesting_test(void **state) {
     assert_true(RationalTime_equal(stack_transformed_time_fifty_clip, middle));
     assert_true(RationalTime_equal(stack_transformed_time_ninetynine_clip, last));
 
-    int num_wrappers = 10;
+    #define num_wrappers 10
```

Comment:
Does MSVC allow `const int num_wrappers = 10;`?

Reply:
No, it does not allow it (and I can't understand why).

Reply:
I *think* but have not verified, that the explanation here might help: https://cplusplus.com/forum/beginner/285003/